processor/filterprocessor/config.go

```diff
@@ -161,6 +168,7 @@ var severityToNumber = map[string]plog.SeverityNumber{
 var errInvalidSeverity = errors.New("not a valid severity")
 
 // logSeverity is a type that represents a SeverityNumber as a string
+// Deprecated: [v0.72.0] use LogConditions instead
```

Comment:
I would add to this comment the version we plan on removing this in, to prevent someone from removing it too quickly :)